DESCRIPTION

```diff
@@ -1,9 +1,9 @@
 Package: extraInserts
 Title: RStudio Addin to Insert Various (Pipe) Operators
 Version: 0.0.0.9000
-Authors@R: as.person(c(
-    "Konrad Zdeb <konrad.zdeb@me.com> [aut, cre]", 
-    "Salim Brüggemann <salim-b@pm.me> [aut]"
+Authors@R: c(
+    person("Konrad", "Zdeb", role = c("aut", "cre"), email = "konrad.zdeb@me.com"),
+    person("Salim", "Brüggemann", role = "aut", email = "salim-b@pm.me", comment = c(ORCID = "0000-0002-5329-5987")),
   ))
```

Comment:
Build failed as the `DESCRIPTION` file appears to be malformed. I think it's just a minor error with a superfluous bracket at the end...
```suggestion
    person("Salim", "Brüggemann", role = "aut", email = "salim-b@pm.me", comment = c(ORCID = "0000-0002-5329-5987")))
```